compiler/parser/src/token.rs

```diff
@@ -6,12 +6,28 @@ use std::fmt::{self, Write};
 /// Python source code can be tokenized in a sequence of these tokens.
 #[derive(Clone, Debug, PartialEq)]
 pub enum Tok {
-    Name { name: String },
-    Int { value: BigInt },
-    Float { value: f64 },
-    Complex { real: f64, imag: f64 },
-    String { value: String, kind: StringKind },
-    Bytes { value: Vec<u8> },
+    Name {
+        name: String,
+    },
+    Int {
+        value: BigInt,
+    },
+    Float {
+        value: f64,
+    },
+    Complex {
+        real: f64,
+        imag: f64,
+    },
+    String {
+        value: String,
+        kind: StringKind,
+        triple_quoted: bool,
+        prefix: Option<String>,
```

Comment:
Added `triple_quoted` and `prefix` to get the number of characters preceding the f-string content.

```
  fr"""abc"""
  -----
    What is the number of characters preceding the f-string content?
```